code/cms/DMSDocumentAddController.php

```diff
@@ -146,6 +144,8 @@ public function getEditForm($id = null, $fields = null)
         $form->Backlink = $backlink;
         // Don't use AssetAdmin_EditForm, as it assumes a different panel structure
         $form->setTemplate($this->getTemplatesWithSuffix('_EditForm'));
+        $form->Fields()->push( new HiddenField('ID', false, $documentSet->ID));
+        $form->Fields()->push(new HiddenField('DSID', false, $documentSet->ID));
```

Comment:
Is this the change you said resolves https://github.com/silverstripe/silverstripe-dms/issues/120?